docs/meshstack.gcp.metering.md

```diff
@@ -3,5 +3,11 @@ id: meshstack.gcp.metering
 title: Metering
 ---
 
-> Detailed Operator Documentation for Google Cloud Platform is under construction. Until we're done, please don't hesitate to get in touch.
+meshStack imports metering data from [GCP Cloud Billing data BigQuery Export](https://cloud.google.com/billing/docs/how-to/export-data-bigquery)
+
+Once billing export has been setup as explained in the GCP documentation, meshStack should be configured with the credentials of a GCP service user that has permission to access the exported billing dataset.
+This service user should also have the permission to run jobs. From the [predefined roles](https://cloud.google.com/bigquery/docs/access-control) these permissions are available in
+`roles/bigquery.jobUser` and `roles/bigquery.dataViewer`. The credentials should be exported in JSON format as mentioned in the [GCP Documentation](https://cloud.google.com/docs/authentication/production).
+
+Additionally, meshStack requires the full path to the BigQuery table where the billing data is stored, so that it can be queried to fetch the billing data.
```

Comment:
please also post the dhall config model - this clarifies the exact parameters required.

Reply:
You mean something like 

> The config model is as follows
`{ platform : Text, bigqueryTable : Text, credentials : Secret }`
?
Not sure if that would help the user though? For example they wouldn't have a clear understanding of what a `Secret` is?

Reply:
yes this is fine, configs with secrets we show as lambdas, see: https://docs.meshcloud.io/docs/meshstack.aws.index.html#configuration-reference

Reply:
and have a look at https://github.com/meshcloud/meshcloud-docs/blob/develop/CONTRIBUTING.md

Reply:
We might have to change the config so will update later.